### PR TITLE
Allow listing and deleting functions

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- add new domain in cbs
- add functions query
- add deleteFunction mutation

**Related issue(s)**
https://github.com/kyma-project/console/issues/1275

### CODE REVIEW

File: components/console-backend-service/internal/domain/serverless/convert/function_test.go [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestConvert_FunctionToGQL(t *testing.T) {
+	t.Run("Success", func(t *testing.T) {
+		expectedName := "expedtedName"
+		expectedNamespace := "expectedNamespace"
+		expectedRuntime := "expectedRuntime"
+		expectedSize := "expectedSize"
+		expectedLabels := gqlschema.Labels{"test": "label"}
```
[the remark this answers is not in the document]
Comment (akucharska) [reply components/console-backend-service/internal/domain/serverless/convert/function_test.go]: Done

File: components/console-backend-service/README.md [hunk 2]
```diff
@@ -97,4 +97,6 @@ go test ./...
 
 ### Verify the code
 
-To check if the code is correct and you can push it, run the `before-commit.sh` script. It builds the application, runs tests, checks the status of the vendored libraries, runs the static code analysis, and ensures that the formatting of the code is correct.
+To check if the code is correct and you can push it, use the `make` command. It builds the application, runs tests, checks the status of the vendored libraries, runs the static code analysis, and checks if the formatting of the code is correct. 
+
+Command `make format` can be used to automatically format incorrect code.
```
Comment (klaudiagrz) [inline components/console-backend-service/README.md]:
```suggestion
To automatically format the incorrect code, use the `make format` command.
```

**Pull request merged**